.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,26 @@
+name: Docker Image CI
+
+on:
+  workflow_dispatch:
```

Comment:
should that not be on merge as well ?

Reply:
modifications afterwards I want to get it into master for testing